src/applications/vaos/tests/e2e/page-objects/AppointmentList/AppointmentListPageObject.js

```diff
@@ -7,9 +7,12 @@ export class AppointmentListPageObject extends PageObject {
   }
 
   selectListItem() {
-    cy.findByTestId('appointment-list-item')
-      .first()
-      .click({ waitForAnimations: true });
+    // get the first appointment list item
+    cy.get(
+      '[data-testid="appointment-list-item"] > :nth-child(1) > :nth-child(1) > :nth-child(1)',
+    )
+      .should('exist')
+      .click();
```

Comment:
Why the change? Was a test failing?

Reply:
Good question. The test was flaky taking up resources. I was trying to isolate the specific element into one line.  I'll revert the code back to using `findBytestId` since it is easier to read. The resource hog was actually coming from `cy.get('h2')`. It was returning too many h2 including those from the header and footer.